sdk/burdock/query/sql/reader/spark_reader.py

```diff
@@ -0,0 +1,74 @@
+import os
+import re
+
+from burdock.metadata.name_compare import BaseNameCompare
+from .rowset import TypedRowset
+
+from burdock.query.sql.ast.tokens import Literal
+from burdock.query.sql.ast.expressions.numeric import BareFunction
+
+class SparkReader:
+    def __init__(self, host, database, user, password=None, port=None):
+        from pyspark.sql import SparkSession
+        self.api = SparkSession.builder.config("spark.rpc.askTimeout", "600s").config("spark.executor.memory", "6g").config("spark.driver.memory", "6g").config("spark.executor.memoryOverhead", "600").config("spark.driver.memoryOverhead", "600").getOrCreate()
```

Comment:
We can check this PR as is but will need to update it for configuration support

Reply:
Oh, shoot, that's a bug.  I wanted to move that out and have the caller pass in the context (like Dylan's sample in email earlier today).  Will fix and update

Reply:
passing in session solved this